.github/workflows/release-native.yml

```diff
@@ -16,18 +16,17 @@ permissions:
 
 jobs:
   scala-native:
-    if: github.event_name == 'release'
```

Comment:
we need to keep this, this workflow supports a dispatch event as well which only runs dockerize.

Reply:
Sure, I am just testing it now 😅